internal/cluster/cluster_test.go

```diff
@@ -2138,7 +2140,7 @@ var _ = Describe("insufficient_state", func() {
 			BaseDNSDomain:   "test.com",
 			PullSecretSet:   true,
 		}}
-		mockEvents.EXPECT().AddEvent(gomock.Any(), gomock.Any(), gomock.Any(), gomock.Any(), gomock.Any(), gomock.Any()).AnyTimes()
+		mockEvents.EXPECT().SendClusterEvent(gomock.Any(), gomock.Any()).AnyTimes()
```

Comment:
Can this expectation be reduced to use eventMatcher to verify an expected event was emitted in this context ?

Reply:
done